group_vars/maintenance.yml

```diff
@@ -288,7 +288,19 @@ telegraf_plugins_extra:
       - timeout = "60s"
       - data_format = "influx"
       - interval = "2m"
-
+  galaxy_job_radar_visualization_stats:
+    plugin: "exec"
+    config:
+      - commands = [
+        "{{ custom_telegraf_env }} /usr/bin/galaxy_failed_jobs_by_destination.sh",
+        "{{ custom_telegraf_env }} /usr/bin/galaxy_most_used_tools_by_destination.sh",
+        "{{ custom_telegraf_env }} /usr/bin/galaxy_num_anonymous_jobs_by_destination.sh",
+        "{{ custom_telegraf_env }} /usr/bin/galaxy_unique_users_job_count_by_destination.sh",
+        "{{ custom_telegraf_env }} /usr/bin/galaxy_longest_running_jobs_by_destination.sh",
+        ]
+      - timeout = "240s"
+      - data_format = "influx"
+      - interval = "1h"
```

Comment:
Those queries are very expensive arn't they? Do we really need to run them this often?

Reply:
No, The SQL queries I have added are optimized as much as possible (I think when I timed their executions, they completed in less than a few seconds (Yes, a few seconds might be an expensive operation)). The interval depends on how often we want the data in the Galaxy Job Radar to be updated for visualization and how "real" the "real-time" data should be for that.

Reply:
Stefan and me have the suspicion that one of our many queries for Grafana is causing our DB problems.
I'm just afraid of we add more of those. Please merge this in and observe how it is going, but my maybe we should also:

* increase the interval
* maybe we need to add new indices somewhere
* maybe we need to fill larger influx-tables and then do the selection in the influx side.

Reply:
I like dumping the data into the influx tables and then processing it there. Let me see the possibilities.

Reply:
Since the data needs to be "real-time" for the visualization, dumping the metrics from Galaxy DB to InfluxDB once a day is not a feasible option. I will rerun each script/query in this PR and update the time it takes in this PR for reference, and then we can proceed with merging and deploying this, as the GJR development seems to be affected because of this.